myhpom/views/content.py

```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+from django.views.decorators.http import require_http_methods
+
+
+@require_http_methods(['GET'])
+def why_plan(request):
+    return render(request, 'myhpom/content/why-plan.html')
```

Comment:
Would it make sense to generalize this a little? Create a single view that takes a URL parameter, and have the parameter indicate the template to use. Then all the static pages would use the same url definition, and we could reverse it this one with something like reverse('myhpom:content', 'why-plan')